src/watchful/client.py

```diff
@@ -1371,21 +1371,6 @@ def combine(c: Any):
     )
 
 
-def exit_backend() -> None:
```

Comment:
We also need to update [examples/api_intro.ipynb](https://github.com/Watchfulio/watchful-py/blob/c935569d1052d1dd7b76a34517dedd58556048ec/examples/api_intro.ipynb#L4664) to remove references to `exit_backend`